flybywiresim/aircraft
# fix: AOC INIT/REVIEW page - ON and OFF time logic.

Partially Fixes #8369

## Summary of Changes

Fixed condition under if (!this.aocTimes.on) to ensure proper triggering of landing block time. 
## Screenshots (if necessary)
Fixed the OFF and ON time discrepancy. 
![Microsoft Flight Simulator Screenshot 2024 05 27 - 14 43 18 91](https://github.com/flybywiresim/aircraft/assets/36414241/97f7e06b-7bde-4df5-8a1b-641c70538d8e)




## References





## Additional context



Discord username (if different from GitHub): rahulrk

## Testing instructions

1. Load a flight plan in simbrief
2. Import flight plan into EFB
3. Go to the AOC INIT/REVIEW page, and hit the INIT LSK.
4. Perform the takeoff. Observe the OFF time (Refresh the AOC INIT/REVIEW page by cycling pages).
5. Perform the landing. The ON time field will populate once the aircraft gears make contact. 

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
